e2e/dractions/disapp.go

```diff
@@ -25,6 +25,9 @@ func EnableProtectionDiscoveredApps(ctx types.Context) error {
 	placementName := name
 	drpcName := name
 
+	// Protects app deployed on DRCluster(c1)
```

Comment:
Here we must add a note:

    // We assume that the application is on the first cluster (c1). This correct only
    // for the current test flow.
    // TODO: must discover where the application is located instead.